[2024-12-29T00:37:02.811+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-29T00:37:02.868+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_sales_analysis_test.extract_sales_data scheduled__2024-12-28T00:00:00+00:00 [queued]>
[2024-12-29T00:37:02.887+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_sales_analysis_test.extract_sales_data scheduled__2024-12-28T00:00:00+00:00 [queued]>
[2024-12-29T00:37:02.888+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-29T00:37:02.929+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_sales_data> on 2024-12-28 00:00:00+00:00
[2024-12-29T00:37:02.951+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=812) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-29T00:37:02.955+0000] {standard_task_runner.py:72} INFO - Started process 821 to run task
[2024-12-29T00:37:02.957+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_sales_analysis_test', 'extract_sales_data', 'scheduled__2024-12-28T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/ecommerse_etl.py', '--cfg-path', '/tmp/tmpdodeswzs']
[2024-12-29T00:37:02.960+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask extract_sales_data
[2024-12-29T00:37:03.162+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_sales_analysis_test.extract_sales_data scheduled__2024-12-28T00:00:00+00:00 [running]> on host 23e56c303c93
[2024-12-29T00:37:03.474+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_sales_analysis_test' AIRFLOW_CTX_TASK_ID='extract_sales_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-28T00:00:00+00:00'
[2024-12-29T00:37:03.478+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-29T00:37:03.479+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-29T00:37:03.481+0000] {logging_mixin.py:190} INFO - Current task name:extract_sales_data state:running start_date:2024-12-29 00:37:02.869750+00:00
[2024-12-29T00:37:03.481+0000] {logging_mixin.py:190} INFO - Dag name:etl_sales_analysis_test and current dag run status:running
[2024-12-29T00:37:03.482+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-29T00:37:03.545+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-29T00:37:03.582+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-29T00:37:03.583+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_sales_analysis_test, task_id=extract_sales_data, run_id=scheduled__2024-12-28T00:00:00+00:00, execution_date=20241228T000000, start_date=20241229T003702, end_date=20241229T003703
[2024-12-29T00:37:03.677+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-29T00:37:03.678+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-29T00:37:03.679+0000] {logging_mixin.py:190} INFO - Dag name:etl_sales_analysis_test queued_at:2024-12-29 00:36:46.792155+00:00
[2024-12-29T00:37:03.681+0000] {logging_mixin.py:190} INFO - Task hostname:23e56c303c93 operator:PythonOperator
[2024-12-29T00:37:03.727+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-29T00:37:03.792+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-29T00:37:03.798+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
